go/src/koding/klientctl/main.go

```diff
@@ -743,6 +744,26 @@ error opening: %s
 					},
 				},
 			}, {
+				Name:        "sync",
+				Usage:       "Manage mount synchronization.",
+				Description: cmdDescriptions["mount-sync"],
+				Action:      ctlcli.ExitErrAction(MachineSyncMount, log, "sync"),
+				Flags: []cli.Flag{
+					cli.DurationFlag{
+						Name:  "timeout, t",
+						Usage: "Maximum time to wait.",
+						Value: time.Minute,
+					},
+					cli.BoolFlag{
+						Name:  "pause",
```

Comment:
> If we have ./pause directory, calling kd mount sync pause could indicate either we 

I think we can require that relative paths must begin with `./` or `../`, go tool works that way so there's no ambiguity if the argument is a package name or directory.

> If #11068 is done, calling kd mount sync pause may be either pausing synchronization for mount in WD or pausing synchronization for mount with name pause

This can be handled at mount name generator phase - add list of reserved mount names and generate one that does not exist in the list.